ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -374,10 +374,10 @@ def test_ipa_healthcheck_fips_enabled(self):
         if (
             parse_version(healthcheck_version) < parse_version("0.17")
             and osinfo.id == 'rhel'
-            and osinfo.version_number == (10,0)
+            and osinfo.version_number >= (10,1)
```

Comment:
Hi @menonsudhir 
we need to have >= (10,0) as the ipa-healthcheck version shipped with rhel 10.0 does not have the patch for fips mode.